includes/class-klp-wc-payment-gateway-blocks-support.php

```diff
@@ -73,24 +73,4 @@ public function get_payment_method_data()
             'logo_urls'   => [$this->gateway->icon],
         ];
     }
-
-    /**
-     * Add failed payment notice to the payment details.
-     *
-     * @param PaymentContext $context Holds context for the payment.
-     * @param PaymentResult $result Result object for the payment.
-     */
-    public function failed_payment_notice(PaymentContext $context, PaymentResult &$result)
```

Comment:
Why are we removing the notice?

Reply:
It wasn't applicable. The code is redundant.